js/piwik.js

```diff
@@ -4960,6 +4967,10 @@ if (typeof window.Matomo !== 'object') {
                 linkTrackingInstalled = false;
                 linkTrackingEnabled   = false;
             };
+            this.disableBrowserFeatureDetection = function ()
```

Comment:
I would assume that we need to add that new method to the list of methods applied first here:
https://github.com/matomo-org/matomo/blob/41b79fe4dbd179338df22c90f2a44aa7c63b4183/js/piwik.js#L7109

Am I right @tsteur?

Reply:
@sgiehl yes definitely 👍  we'd want to execute that method quite early in the process to make sure we don't access the browser features. We probably want to execute it before `disableCookies` or after. The earlier the better I guess.